properties.schema

```diff
@@ -19,20 +19,12 @@
                 "_isEnabled": {
                   "type":"boolean",
                   "required":true,
+                  "default": true,
                   "title":"Is Enabled",
                   "inputType": {"type": "Boolean", "options": [true, false]},
                   "validators": [],
                   "help": "If set to 'true', this course will be tracked by TinCan/xAPI"
                 },
-                "_activityID": {
```

Comment:
In cmi5, it is a requirement to pass the activityID in via the launch url (https://github.com/AICC/CMI-5_Spec_Current/blob/quartz/cmi5_spec.md#81-launch-method). This is not a requirement for xAPI (but is one of the more common methods)

That being said, 'The LMS MUST place the value for activityId in the query string based on the definition of the AU in the course structure.'

We will still need this in the schema for both xAPI and cmi5.